ckanext-notify/README.rst

```diff
@@ -0,0 +1,166 @@
+.. You should enable this project on travis-ci.org and coveralls.io to make
+   these badges work. The necessary Travis and Coverage config files have been
+   generated for you.
+
+.. image:: https://travis-ci.org/"WNjihia"/ckanext-notify.svg?branch=master
+    :target: https://travis-ci.org/"WNjihia"/ckanext-notify
+
+.. image:: https://coveralls.io/repos/"WNjihia"/ckanext-notify/badge.svg
+  :target: https://coveralls.io/r/"WNjihia"/ckanext-notify
+
+.. image:: https://pypip.in/download/ckanext-notify/badge.svg
+    :target: https://pypi.python.org/pypi//ckanext-notify/
+    :alt: Downloads
+
+.. image:: https://pypip.in/version/ckanext-notify/badge.svg
+    :target: https://pypi.python.org/pypi/ckanext-notify/
+    :alt: Latest Version
+
+.. image:: https://pypip.in/py_versions/ckanext-notify/badge.svg
+    :target: https://pypi.python.org/pypi/ckanext-notify/
+    :alt: Supported Python versions
+
+.. image:: https://pypip.in/status/ckanext-notify/badge.svg
+    :target: https://pypi.python.org/pypi/ckanext-notify/
+    :alt: Development Status
+
+.. image:: https://pypip.in/license/ckanext-notify/badge.svg
+    :target: https://pypi.python.org/pypi/ckanext-notify/
+    :alt: License
+
+=============
+ckanext-notify
+=============
+
+.. Put a description of your extension here:
+   What does it do? What features does it have?
+   Consider including some screenshots or embedding a video!
+
+
+------------
+Requirements
+------------
+
+For example, you might want to mention here which versions of CKAN this
+extension works with.
+
+
+------------
+Installation
+------------
+
+.. Add any additional install steps to the list below.
+   For example installing any non-Python dependencies or adding any required
+   config settings.
+
+To install ckanext-notify:
+
+1. Activate your CKAN virtual environment, for example::
+
+     . /usr/lib/ckan/default/bin/activate
+
+2. Install the ckanext-notify Python package into your virtual environment::
+
+     pip install ckanext-notify
+
+3. Add ``notify`` to the ``ckan.plugins`` setting in your CKAN
+   config file (by default the config file is located at
+   ``/etc/ckan/default/production.ini``).
+
+4. Restart CKAN. For example if you've deployed CKAN with Apache on Ubuntu::
+
+     sudo service apache2 reload
+
+
+---------------
+Config Settings
+---------------
+
+Document any optional config settings here. For example::
+
+    # The minimum number of hours to wait before re-checking a resource
+    # (optional, default: 24).
+    ckanext.notify.some_setting = some_default_value
```

Comment:
I think this can be more specific, what are the configs expected for ckanext-notify, unless there is one or two specific settings, why not explicitly state them e.g the file on which they should be updated, and the settings keys and values, otherwise instead of having a vague configs, we can ignore this section altogether